migrations/sql/000021_add_mod_platform.up.sql

```diff
@@ -0,0 +1,21 @@
+-- Mod Links
+create table if not exists mod_links
+(
+    id                      varchar(14) not null constraint mod_links_pkey primary key,
+    mod_version_link_id     varchar(14),
+    platform                varchar(16),
+    key                     text,
+    hash                    char(64),
+    size                    bigint
+);
+create index if not exists idx_mod_links_id on mod_links (mod_version_link_id, platform);
+
+-- SML Links
+create table if not exists sml_links
```

Comment:
Please rename to `sml_archs`

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/8ea4f2b30882e41af00e95ce37ab41b3a4ebd4e3